### DISCUSSION TITLE
Monitor SSM server itself

### DISCUSSION THREAD

**Discussion by chopeta** [SquirrelCorporation/SquirrelServersManager#40] [Q&A]
How would we go about monitoring SSM server itself?

Should this be enabled automatically?

Should we add a new device using "localhost" or the IP of the server?

Actually, while trying this, the add new device popup doesn't allow me to set "localhost":

![image](https://github.com/SquirrelCorporation/SquirrelServersManager/assets/7503721/a512ada2-a31c-4a6b-bdf3-4c07ef4bd0b2)

**Comment by SquirrelDeveloper** [comment]
Yes, the regexp/validation part is quite restrictive. Did you try 127.0.0.1 though?
